src/main/kotlin/no/nav/medlemskap/services/pdl/PdlClient.kt

```diff
@@ -160,6 +161,13 @@ class PdlService(private val pdlClient: PdlClient, private val clusterName: Stri
         return PdlMapper.mapTilFoedselsaar(pdlClient.hentFoedselsaar(fnr, callId))
     }
 
+    // TODO: feilhåndtering; hva vil vi gjøre dersom det ikke finnes arbeidsgiver?
+    suspend fun hentStatsborgerskap(fnr: String, callId: String): List<Statsborgerskap> {
+        return pdlClient.hentNasjonalitet(fnr, callId).data?.hentPerson?.statsborgerskap?.map { PdlMapper.mapStatsborgerskap(it) }
+                .let {
```

Comment:
Jeg er litt usikker på denne her. Siden jeg ikke er så veldig kjent med .let, så vil jeg bare spørre. Kjører .let kun på typer som ikke er null, eller på typer som er null?

Reply:
let kjører uavhengig av om verdien er null eller ikke. Men det er vanlig å bruke let sammen med safe-class operator ? for at den kodeblokken kun kjøres dersom verdien ikke er null.

Reply:
Kanskje det kunne passet å ta ut listen i egen variabel, og heller skrive .ifEmpty { //log } før listen med statsborgerskap returneres